src/Plugin/views/filter/Section.php

```diff
@@ -253,13 +282,21 @@ public function query() {
       if (!empty($this->options['section_filter']['show_hierarchy'])) {
         $values = $this->getChildren($values);
       }
+      // @TODO: This is probably correct, because user data is stored with
```

Comment:
So this is the part I'm, a bit stuck on. The filters use the section_id, not the entity_id that is stored in the `section_association__user_id` table. I couldn't make those JOINs work in Views, so instead I am rewriting the query parameters.

This seems brittle and prone to errors but I don't see another obvious solution to the problem.